.travis.yml

```diff
@@ -8,15 +8,15 @@ php:
 
 matrix:
   include:
+    - php: hhvm
```

Comment:
+1 more reason to move HHVM to separate CI. 
I will try to work on Wercker to support all current test cases and then we could try to move it out. 
I think moving that away would improve the overall stability of Travis builds